src/lib/jsonParseV2/parseEntity.cpp

```diff
@@ -79,17 +79,18 @@ std::string parseEntity(ConnectionInfo* ciP, Entity* eP, bool eidInURL)
     {
       LM_W(("Bad Input (No entity id specified"));
       eP->errorCode.fill("ParseError", "no entity id specified");
-      ciP->httpStatusCode = SccBadRequest;;
+      ciP->httpStatusCode = SccBadRequest;
+
       return eP->render(ciP, EntitiesRequest);
     }
   }
-  else if (document.Empty()) 
+  else if (document.ObjectEmpty()) 
   {
     LM_W(("Bad Input (Empty payload)"));
     eP->errorCode.fill("ParseError", "empty payload");
-    ciP->httpStatusCode = SccBadRequest;;
-    return eP->render(ciP, EntitiesRequest);
+    ciP->httpStatusCode = SccBadRequest;
```

Comment:
I understand that from now on all the changes in this file are style things (probably related with traces insertion/removal). I see several empty lines... not sure if that is intentional.